torchtext/experimental/datasets/raw/question_answer.py

```diff
@@ -57,32 +57,59 @@ def __iter__(self):
                 break
 
 
-def _setup_datasets(dataset_name, root='.data'):
+def _setup_datasets(dataset_name, root='.data', data_select=('train', 'dev')):
```

Comment:
There is no test?

Reply:
We use train, valid and test across most datasets. Do we want to add a note here or take the liberty of not using 'dev' but 'test' instead?

Reply:
We have "test_squad1" and "test_squad2" tests in "test_builtin_datasets.py" file.